bigframes/core/pyformat.py

```diff
@@ -91,6 +102,8 @@ def pyformat(
     sql_template: str,
     *,
     pyformat_args: dict,
+    session: bigframes.session.Session,
```

Comment:
Need to support the case where `session` is `None` for https://github.com/googleapis/python-bigquery-dataframes/pull/1802 support. If this is the first SQL Cell / bigframes code run in a notebook, we need to run pyformat to try and auto-detect a location.

Reply:
Done.